blocks/article-hero-video/article-hero-video.js

```diff
@@ -101,10 +101,12 @@ function buildVideoTag(url) {
   video.style.width = '100%';
   video.style.height = '100%';
   video.classList.add('article-hero-video-element');
-  video.setAttribute('preload', 'auto');
-  video.setAttribute('autoplay', '');
-  video.setAttribute('muted', '');
-  video.setAttribute('loop', '');
+
+  // Set boolean attributes
+  video.muted = true;
+  video.autoplay = true;
+  video.loop = true;
+  video.setAttribute('muted', 'true');
```

Comment:
Muted set twice, and preloading not set. Is this the intention?

Reply:
Preload did not have any effect. 

About muted: You can see with the current muted setting, the video in main branch **only** plays in safari, not in chrome, edge
https://main--24life--hlxsites.hlx.page/drafts/ashrigondeka/nutritionist-kelly-leveques-fab-four-to-get-in-shape

for firefox, this user-level setting should have played it, but it did not. It only plays when audio and video is allowed.  

<img width="621" alt="image" src="https://github.com/hlxsites/24life/assets/26192212/ae29c884-517b-48fc-a102-5064b85783b7">